lib/main.dart

```diff
@@ -29,6 +29,7 @@ class _RandomWordsState extends State<RandomWords> {
   final _suggestions = <WordPair>[];
   final _saved = <WordPair>{};
   final _biggerFont = const TextStyle(fontSize: 18.0);
+  var _title = Text("Cnange me");
```

Comment:
in state, it's better to keep `_title` as just a string (not `Text` widget). Then when state changes, you change this string to another string, and in `build` you do `title: Text(_title)` instead of `title: _title`.